httpload/cmd/httpload/main.go

```diff
@@ -0,0 +1,440 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"sort"
+	"strings"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/pkg/errors"
```

Comment:
I'm not sure you really want to use this package. The go standard library now has added to https://godoc.org/errors many of the features of the package you imported

Reply:
Maybe it can help: the `noise` is generated by the `make test`  command, but I do not know why.

Reply:
Just write the import yourself. replace `github.com/pkg/errors` with just `errrors`.
`gofmt` only adds/removes package but never overwrites one so when it's confused, just write it yourself